src/App.vue

```diff
@@ -37,71 +71,80 @@ export default {
   data() {
     return {
       showModal: false,
-    }
+    };
   },
   watch: {
     theme(value) {
-      document.documentElement.classList.remove('dark', 'light', 'october')
-
-      switch(value) {
-        case 'dark':
-          document.documentElement.classList.add('dark')
-          break
-        case 'light':
-          document.documentElement.classList.add('light')
-          break
-        case 'october':
-          document.documentElement.classList.add('dark', 'october')
-          break
+      document.documentElement.classList.remove("dark", "light", "october");
+
+      switch (value) {
+        case "dark":
+          document.documentElement.classList.add("dark");
+          break;
+        case "light":
+          document.documentElement.classList.add("light");
+          break;
+        case "october":
+          document.documentElement.classList.add("dark", "october");
+          break;
+        case "auto dark":
+          document.documentElement.classList.add("auto dark");
+          break;
+        case "auto light":
+          document.documentElement.classList.add("auto light");
+
         default:
-          document.documentElement.classList.add('dark')
-          break
+          document.documentElement.classList.add("dark");
+          break;
       }
     },
```

Comment:
This entire method can be updated to call the new method I recommend in a comment below.

```js
    theme(_value) {
      this.updateTheme()
    },
```